### PR TITLE
Enables and disables services

### LINKED ISSUE #920: Possibilidade de ativar e inativar um serviço
## Feature Description

Precisamos que assim como as tecnologias, exista a possibilidade de ativar e inativar um serviço.

---------------

## Acceptance criteria

* Poder ativar ou inativar um serviço

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by alexandreadames**
## Summary

Resolves #920 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/api/app/Controllers/Http/ServiceController.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ class ServiceController {
 	async index({ request }) {
 		const filters = request.all();
 		return Service.query()
+			.where({ active: true })
```
Comment (mateus4k) [inline packages/api/app/Controllers/Http/ServiceController.js:16]: Put this condition inside the model. These changes will be useful when we need to extend the service available logic.

File: packages/api/app/Controllers/Http/TechnologyController.js [hunk 2]
```diff
@@ -590,8 +590,10 @@ class TechnologyController {
 			'technologyCosts.costs',
 		]);
 
-		if (technology.status === technologyStatuses.PUBLISHED) {
+		if (technology.active) {
```
Comment (mateus4k) [inline packages/api/app/Controllers/Http/TechnologyController.js:593]: The same should happen when the technology is published or not, right?

Comment (alexandreadames) [reply packages/api/app/Controllers/Http/TechnologyController.js:593]: Sim, mas nessa rota não cabe essa verificação

**Review by mateus4k — APPROVED** [review]
👍

**Pull request merged**